public/index.html

```diff
@@ -24,6 +24,16 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <script src="https://code.jquery.com/jquery-3.4.1.js"></script>
+    <script type="text/javascript">
+      var zero = 0;
+      $(document).ready(function() {
```

Comment:
what is this for?
why add jquery? you can just put your javascript code in some hooks right?